src/funding/funding.ts

```diff
@@ -149,6 +147,12 @@ export class Funding {
    * @param transaction
    */
   public withdraw = async(transaction: IDaoTransaction): Promise<void> => {
```

Comment:
can you define this function in our normal typescript way?

Reply:
Done